### PR TITLE
[WIP] Ensure omnia metapackage is built for each Python version

### PR DESCRIPTION
**Pull request by jchodera**
Testing this as a solution to issues observed in #103

### CODE REVIEW

**Comment by jchodera** [comment]
I've tested that this creates versions of the omnia metapackage resolved by Python and numpy version on my machine.  I have not yet tested the rebuilding or installing of *all* packages.

File: omnia/meta.yaml [hunk 1]
```diff
@@ -4,7 +4,12 @@ package:
 
 
 requirements:
+  build:
+    - python # necessary to ensure packages are built for each Python version
+    - numpy # necessary to ensure packages are built for each Python version
```
Comment (rmcgibbo) [inline omnia/meta.yaml]: Are you sure that the numpy versioning is necessary?

**Comment by rmcgibbo** [comment]
@jchodera: I think I fixed the issue by removing some wrong packages from a month ago on binstar/omnia/pymbar that didn't declare a dependency on python.

```
$ conda create -n 272 python=2.7
$ conda install omnia
Fetching package metadata: ....
Solving package specifications: .
Package plan for installation in environment /Users/rmcgibbo/miniconda/envs/272:

The following packages will be downloaded:

    package                    |            build
    ---------------------------|-----------------
    pdbfixer-1.1               |       np19py27_0         158 KB
    omnia-1.0                  |                0           1 KB
    ------------------------------------------------------------
                                           Total:         159 KB

The following NEW packages will be INSTALLED:

    dateutil:     2.1-py27_2
    docutils:     0.12-py27_0
    fftw3f:       3.3.3-1
    hdf5:         1.8.14-0
    jinja2:       2.7.3-py27_1
    markupsafe:   0.23-py27_0
    mdtraj:       1.2.0-np19py27_0
    msmbuilder:   3.0.0-np19py27_0
    nose:         1.3.4-py27_0
    numexpr:      2.3.1-np19py27_0
    numpy:        1.9.1-py27_0
    numpydoc:     0.5-py27_0
    omnia:        1.0-0
    openmm:       6.2-py27_1
    pandas:       0.15.2-np19py27_0
    pdbfixer:     1.1-np19py27_0
    pygments:     1.6-py27_0
    pymbar:       2.1beta0-np19py27_0
    pytables:     3.1.1-np19py27_2
    pytz:         2014.9-py27_0
    scikit-learn: 0.15.2-np19py27_0
    scipy:        0.15.0-np19py27_0
    scripttest:   1.3-py27_0
    setuptools:   11.3.1-py27_0
    six:          1.9.0-py27_0
    sphinx:       1.2.3-py27_0

Proceed ([y]/n)? y

Fetching packages ...
pdbfixer-1.1-n 100% |##############################################################################################################| Time: 0:00:00 479.03 kB/s
omnia-1.0-0.ta 100% |##############################################################################################################| Time: 0:00:00 120.10 kB/s
Extracting packages ...
[      COMPLETE      ] |################################################################################################################################| 100%
Linking packages ...
[      COMPLETE      ] |################################################################################################################################| 100%
```

**Comment by jchodera** [comment]
We believe this is solved now. Closing!

**Pull request closed without merging**